.plan

```diff
@@ -16,7 +16,7 @@ Units:
      * ...
 
 
-* NumberLike comes into our project (maybe duplicate in Finance)
+* Quantity comes into our project (maybe duplicate in Finance)
```

Comment:
I think we'll resolve this 'issue' by not using `Quantity` directly in the rest of the DSL. I think we'll just make a member definition of Addable/Subtractale/etc for quantities in the upcoming bridge. That will hopefully provide the flexibility to use the Finance DSL and the Engine/DSL both separately and together without any duplicate code and/or conflicts.